lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
That was fixed in https://github.com/nodejs/node/issues/884, which first appeared in 3.0.0 :).

But, stuff like https://github.com/nodejs/node/issues/2734 is not encouraging.